llvm/lib/Target/AMDGPU/AMDGPUInstructionSelector.cpp

```diff
@@ -5595,6 +5595,12 @@ void AMDGPUInstructionSelector::renderFPPow2ToExponent(MachineInstrBuilder &MIB,
   MIB.addImm(ExpVal);
 }
 
+void AMDGPUInstructionSelector::renderRoundMode(MachineInstrBuilder &MIB,
+                                                const MachineInstr &MI,
+                                                int OpIdx) const {
+  MIB.addImm((MI.getOperand(OpIdx).getImm() + 3) % 4);
```

Comment:
Probably should comment something about the encoding values, like take the comment from the custom lowering

Reply:
Done! Thanks